modules/admission-webhook-types.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-plug-ins.adoc
+
+[id="admission-webhook-types_{context}"]
+= Types of webhook admission plug-ins
+
+Cluster administrators can call out to webhook servers through the mutating admission plug-in or the validating admission plug-in in the API server admission chain.
+
+[id="mutating-admission-plug-in_{context}"]
+== Mutating admission plug-in
+
+The mutating admission plug-in is invoked during the mutation phase of the admission process, which allows modification of resource content before it is persisted. One example webhook that can be called through the mutating admission plug-in is the Pod Node Selector feature, which uses an annotation on a namespace to find a label selector and add it to the pod specification.
+
+[id="mutating-admission-plug-in-config_{context}"]
+.Sample mutating admission plug-in configuration
+
+[source,yaml]
+----
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration <1>
+metadata:
+  name: <webhook_name> <2>
+webhooks:
+- name: <webhook_name> <3>
+  clientConfig: <4>
+    service:
+      namespace: default <5>
+      name: kubernetes <6>
+     path: <webhook_url> <7>
+    caBundle: <ca_signing_certificate> <8>
+  rules: <9>
+  - operations: <10>
+    - <operation>
+    apiGroups:
+    - ""
+    apiVersions:
+    - "*"
+    resources:
+    - <resource>
+  failurePolicy: <policy> <11>
```

Comment:
Add one new line bolded, just like below, effective measured, run into error otherwise.
webhooks:
...
  failurePolicy: <policy> <11>
  **sideEffects: None**

Reply:
Line now added.